tests/common/libvm.sh

```diff
@@ -530,9 +530,14 @@ vm_ostreeupdate_lift_commit() {
 # use a previously stolen commit to create an update on our vmcheck branch,
 # complete with version string and pkglist metadata
 vm_ostreeupdate_create() {
-  branch=$1; shift
+  ver=$1; shift
```

Comment:
Can you expand on this? As long as we only specify branches that were previously used in a `vm_ostreeupdate_lift_commit` operation, it should work. (And if not, we should fix that!)

If we reboot into the original origin tree, then we won't have any of the added packages from the lifted commit. Actually, we won't even be using the rpm-ostree bits we `vmoverlay`ed.

Reply:
>   --tree=ref=$branch --add-metadata-string=version=$branch

I think the core problem is I keep getting confused by this.  Binding the source and the target version together seems wrong.